Solutions/Torq/SolutionMetadata.json

```diff
@@ -0,0 +1,15 @@
+{
+	"publisherId": "azuresentinel",
+	"offerId": "azure-sentinel-solution-torq",
```

Comment:
Please confirm once if this solution is microsoft supported 
In case if not 
please change the publisher id and support details